File: ToyLanguage7/ToyLanguage7/MyList.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace ToyLanguage7
{
    class MyList<T> : MyIList<T>
    {
        private List<T> list;

        public MyList() { this.list = new List<T>(); }

        public void add(T elem)
        {
            list.Add(elem);
        }

        public bool isEmpty()
        {
            return list.Count() == 0;
        }
        
        public int size()
        {
            return list.Count();
        }

        public override string ToString()
        {
            string result = "[";
            foreach (var el in this.list)
                result += el + " ";
            //result = result.Remove(result.Length - 1, 1);
            result += "]";
            return result;
        }
    }
}

File: ToyLanguage8/src/Model/MyList.java
package Model;

import java.util.ArrayList;
import java.util.List;

public class MyList<T> implements MyIList<T> {
    private ArrayList<T> list;

    public MyList() {
        this.list = new ArrayList<>();
    }

    @Override
    public void add(T elem) {
        list.add(elem);
    }

    @Override
    public int size() {
        return list.size();
    }

    @Override
    public boolean isEmpty() {
        return list.size() == 0;
    }

    @Override
    public void remove(int position) {
        list.remove(position);
    }

    @Override
    public ArrayList<String> getOutList() {
        ArrayList<String> getOut = new ArrayList<>();
        list.forEach(el->getOut.add(el.toString()));
        return getOut;
    }

    public List<T> getList() {
        return list;
    }

    @Override
    public String toString() {
        return String.valueOf(list);
    }
}
